app/views/tutors/show.html.erb

```diff
@@ -250,7 +252,9 @@
                           <td> <%= link_to s.name, s %> </td>
                           <td> <%= s.email %> </td>
                           <td> <%= form_tag('remove_student', method: 'patch') do %>
-                            <%= submit_tag('Remove Student') %>
+                            <%= button_tag(type: "submit", class: "btn btn-default") do %>
```

Comment:
Let's stick to the convention of using `btn-danger` for destructive actions like removing a student.